src/main/java/io/confluent/connect/jdbc/TableMonitorThread.java

```diff
@@ -102,7 +104,7 @@ private boolean updateTables() {
     synchronized (db) {
       final List<String> tables;
       try {
-        tables = JdbcUtils.getTables(db);
+        tables = JdbcUtils.getTables(db,table_types);
```

Comment:
I don't think that'll have the result many people expect. The goal is to enable broad copying by default (thus the reason for defaulting to copying all tables). Just considering views, it's not obvious you'd want to copy all views as well. But there are also other types of tables that I can't even think of good reasons for including -- synonym/aliases and system tables, for example.

In any case, at this point we also need to be careful to consider compatibility.